assets/i18n/locales/en.json

```
@@ -17,6 +18,11 @@
   "invites/error_title": "Access Code Issue",
   "invites/error": "There was an issue accepting your Access Code, please try again and/or contact {{email}}",
   "invites/placeholder": "Access Code",
+  "location/edit": "Edit location",
+  "location/delete": "Delete location",
+  "location/errorUpdate": "Edit failed",
+  "location/errorUpdateDescription": "Location on server was newer than the version on your local device. We have discarded your changes and updated the location. If you still wish to make changes, please try again.",
```

Can we tweak the wording to be a little friendlier to less technical folks?

```suggestion
  "location/errorUpdateDescription": "Someone else already updated this location. We have downloaded their update to your device. Please review the update and try your changes again.",
```